Searching & sorting/bubble_sort.py

```diff
@@ -0,0 +1,12 @@
+# Bubble sort
+def bubble_sort(list):
+    for i in range(len(list)):
```

Comment:
add comments for explanation and make separate folder for sorting then try to push